modules/nw-rfhe-installing-operator-cli.adoc

```diff
@@ -101,5 +101,5 @@ $ oc get csv -n openshift-baremetal-events -o custom-columns=Name:.metadata.name
 [source,terminal]
 ----
 Name                                              Phase
-baremetal-event-relay.v4.10.0   Succeeded
+baremetal-event-relay.v4.10.0                     Succeeded
```

Comment:
baremetal-hardware-event-proxy-operator.v4.10.0

The subscription CR needs to reflect the name: Subscription/spec/name should be baremetal-hardware-event-proxy-operator

In my testing I tried openshift-hw-events and openshift-baremetal-events namespace. The operator appeared to install correctly in either but I defer to @yliu127 for what was tested.

The namespace must not have the workload partitioning annotation.

Reply:
@imiller0 did you check the old repo? There is no build to verify yet, so I can't confirm on this atm. But the name from catalog source was supposed to change to these as far as I understand. https://docs.google.com/document/d/19fED5Pm2cUNxmtD0AV4tSoQUfduO7nC3jVzXdt9fHeA/edit?disco=AAAAZ3hXV7E